src/box/errcode.h

```diff
@@ -141,287 +141,313 @@ struct errcode_record {
+	_(ER_COMPLEX_FOREIGN_KEY_FAILED, 240,	"Foreign key constraint '%s' failed: %s", "constraint", STRING, "details", STRING) \
+	_(ER_WRONG_SPACE_UPGRADE_OPTIONS, 241,	"Wrong space upgrade options: %s", "details", STRING) \
+	_(ER_NO_ELECTION_QUORUM, 242,		"Not enough peers connected to start elections: %d out of minimal required %d", "actual", INT, "quorum", INT)\
 	_(ER_SSL, 243,				"%s") \
-	_(ER_SPLIT_BRAIN, 244,			"Split-Brain discovered: %s") \
-	_(ER_OLD_TERM, 245,			"The term is outdated: old - %llu, new - %llu") \
+	_(ER_SPLIT_BRAIN, 244,			"Split-Brain discovered: %s", "details", STRING) \
+	_(ER_OLD_TERM, 245,			"The term is outdated: old - %llu, new - %llu", "old_term", ULLONG, "new_term", ULLONG) \
 	_(ER_INTERFERING_ELECTIONS, 246,	"Interfering elections started")\
 	_(ER_ITERATOR_POSITION, 247,		"Iterator position is invalid") \
-	_(ER_DEFAULT_VALUE_TYPE, 248,		"Type of the default value does not match tuple field %s type: expected %s, got %s") \
-	_(ER_UNKNOWN_AUTH_METHOD, 249,		"Unknown authentication method '%s'") \
-	_(ER_INVALID_AUTH_DATA, 250,		"Invalid '%s' data: %s") \
-	_(ER_INVALID_AUTH_REQUEST, 251,		"Invalid '%s' request: %s") \
-	_(ER_WEAK_PASSWORD, 252,		"Password doesn't meet security requirements: %s") \
-	_(ER_OLD_PASSWORD, 253,			"Password must differ from last %d passwords") \
-	_(ER_NO_SUCH_SESSION, 254,		"Session %llu does not exist") \
-	_(ER_WRONG_SESSION_TYPE, 255,		"Session '%s' is not supported") \
+	_(ER_DEFAULT_VALUE_TYPE, 248,		"Type of the default value does not match tuple field %s type: expected %s, got %s", \
+						"field", STRING, "expected", STRING, "actual", STRING) \
+	_(ER_UNKNOWN_AUTH_METHOD, 249,		"Unknown authentication method '%s'", "auth_method", STRING) \
+	_(ER_INVALID_AUTH_DATA, 250,		"Invalid '%s' data: %s", "auth_method", STRING, "details", STRING) \
+	_(ER_INVALID_AUTH_REQUEST, 251,		"Invalid '%s' request: %s", "auth_method", STRING, "details", STRING) \
+	_(ER_WEAK_PASSWORD, 252,		"Password doesn't meet security requirements: %s", "details", STRING) \
+	_(ER_OLD_PASSWORD, 253,			"Password must differ from last %d passwords", "history_length", INT) \
+	_(ER_NO_SUCH_SESSION, 254,		"Session %llu does not exist", "sid", ULLONG) \
+	_(ER_WRONG_SESSION_TYPE, 255,		"Session '%s' is not supported", "session_type", STRING) \
 	_(ER_PASSWORD_EXPIRED, 256,		"Password expired") \
 	_(ER_AUTH_DELAY, 257,			"Too many authentication attempts") \
 	_(ER_AUTH_REQUIRED, 258,		"Authentication required") \
-	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed for %s") \
-	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s") \
-	_(ER_BOOTSTRAP_NOT_UNANIMOUS, 261,	"Replica %s chose a different bootstrap leader %s") \
-	_(ER_CANT_CHECK_BOOTSTRAP_LEADER, 262,	"Can't check who replica %s chose its bootstrap leader") \
+	_(ER_SQL_SEQ_SCAN, 259,			"Scanning is not allowed for %s", "object", STRING) \
+	_(ER_NO_SUCH_EVENT, 260,		"Unknown event %s", "even", STRING) \
```

Comment:
"value"

Reply:
Fixed on the above comment for "unknown something".